src/vector/index.js

```diff
@@ -30,7 +30,6 @@ require('babel-polyfill');
 
 // CSS requires: just putting them here for now as CSS is going to be
```

Comment:
happy to change the comment, but would defn prefer to understand what @dbkr had in mind in the first place